### PR TITLE
add mdm profile info table

### PR DESCRIPTION
**Pull request by groob**
excuse the exec jank, https://github.com/facebook/osquery/issues/1666#issuecomment-160812564

```
osquery> select * from kolide_mdm_info;
                 enrolled = true
               server_url = https://dev.micromdm.io/mdm/connect
              checkin_url = https://dev.micromdm.io/mdm/checkin
            access_rights = 8191
             install_date = 2017-12-01 22:53:00 +0000
       payload_identifier = com.github.micromdm.micromdm.enroll
                    topic = com.apple.mgmt.External.50C91AEB-9506-484C-A83A-86CBDE06A95F
             sign_message = true
identity_certificate_uuid = f8f9b353-393a-45eb-b70d-51c64910e943
         has_scep_payload = true
```

### CODE REVIEW

File: osquery/mdm.go [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+				continue
+			}
+			if item.PayloadType == "com.apple.mdm" {
+				enrollProfile := item.PayloadContent
+				enrollProfileItems = payload.ProfileItems
+				mdmResults = map[string]string{
+					"enrolled":                  "true",
+					"server_url":                enrollProfile.ServerURL,
+					"checkin_url":               enrollProfile.CheckInURL,
+					"access_rights":             fmt.Sprintf("%d", enrollProfile.AccessRights),
```
Comment (marpaia) [inline osquery/mdm.go]: `strconv.Atoi` if there is no other formatting going on here

Comment (groob) [reply osquery/mdm.go]: strconv.Atoi has an `err` return making it more annoying to since it can't be used on the same line.

Comment (zwass) [reply osquery/mdm.go]: Maybe you mean `strconv.Itoa`?

Comment (groob) [reply osquery/mdm.go]: Ah good call. I'll use `strconv.Itoa`.

File: osquery/mdm.go [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+					"install_date":              payload.ProfileInstallDate,
+					"payload_identifier":        payload.ProfileIdentifier,
+					"sign_message":              fmt.Sprintf("%v", enrollProfile.SignMessage),
```
Comment (zwass) [inline osquery/mdm.go]: If we are preferring `strconv` to `fmt`, I think we should use `strconv.FormatBool` here.

**Pull request merged**